packages/angular/cli/commands/update-impl.ts

```diff
@@ -29,6 +29,7 @@ const oldConfigFileNames = [
 
 export class UpdateCommand extends SchematicCommand<UpdateCommandSchema> {
   public readonly allowMissingWorkspace = true;
+  protected collectionName = '@schematics/update';
```

Comment:
This is intentionally not present.  The command itself handles the options.  It's only a schematic command because there's no other mechanism currently to easily execute a schematic.

Reply:
Removed, PTAL.